Src/Tracers/Zipkin/V2/V1SpanConverter.cs

```diff
@@ -42,66 +42,66 @@ public static V1SpanConverter Create()
         public List<Span> Convert(zipkin4net.Tracers.Zipkin.Span source)
         {
             List<Span> results = new List<Span>();
-            convert(source, results);
+            Convert(source, results);
             return results;
         }
 
-        private void convert(zipkin4net.Tracers.Zipkin.Span source, List<Span> sink)
+        private void Convert(zipkin4net.Tracers.Zipkin.Span source, List<Span> sink)
```

Comment:
According to the naming convention, the first letter of a `private` method should be lowercase. Please pay close attention.

Reply:
No. all methods in C# should be upper case with any modifier. 

https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions

Reply:
> No. all methods in C# should be PascalCasing with any modifier.
> 
> https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions

Maybe you are right, although I don't think making `public` and `private` naming conventions consistent is a good choice. However, I respect everyone's decision.